src/renderer/components/button/button.scss

```diff
@@ -54,8 +54,7 @@
 
   form:not([novalidate]):invalid &[type=submit]:not(.active),
   &:disabled {
-    color: silver;
-    background: $buttonDisabledBackground;
+    opacity: 50%;
```

Comment:
is this Lens standard behaviour?

Reply:
Yes, most other disabled items change opacity.